wagtail/wagtailsearch/backends/elasticsearch.py

```diff
@@ -409,15 +442,33 @@ def _get_es_body(self, for_count=False):
             if sort is not None:
                 body['sort'] = sort
 
+            # Add highlighting into a body, if fields are specified
+            if self.highlight_params.get('fields'):
+                highlight = {
+                    'fields': self.highlight_params['fields'].values(),
```

Comment:
At some point in here we need to translate the field names passed in from the user to the actual column names.

For example: ``body`` may be indexed as ``events_eventpage_body`` and ``home_homepage_body``